app/client/src/components/DropZone.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <div v-bind='getRootProps()' class=dropzone>
+      <input v-bind='getInputProps()' />
+      <p v-if='isDragActive'>Drop the files here ...</p>
+      <p v-else>Drag 'n' drop your fasta files here, or click to select files</p>
```

Comment:
I would probably just say "Drag and drop your fasta...", but that's just my subjective preference!